Sources/Controllers/DashboardOnMap/MapSettings/MapSettingsGpxViewController.swift

```diff
@@ -0,0 +1,592 @@
+                    gpxRow.title = title
+                    gpxRow.setObj(distance, forKey: "distance")
+                    gpxRow.setObj(time, forKey: "time")
+                    gpxRow.setObj(waypointCount, forKey: "waypointCount")
+                    gpxRow.iconName = "ic_custom_trip"
+                    gpxRow.iconTintColor = visibleGpxFilePaths.contains(gpx.gpxFilePath) ? .iconColorActive : .iconColorDisabled
+                }
+            }
+            if isShowingVisibleTracks && !recentlyVisibleGpxList.isEmpty && !isSearchActive {
+                let recentlyVisibleSection = tableData.createNewSection()
+                recentlyVisibleSection.headerText = localizedString("recently_visible") + " (\(recentlyVisibleGpxList.count))"
+                for gpx in recentlyVisibleGpxList {
+                    let title = gpx.getNiceTitle()
+                    let distance = OAOsmAndFormatter.getFormattedDistance(gpx.totalDistance) ?? "N/A"
+                    let time = OAOsmAndFormatter.getFormattedTimeInterval(TimeInterval(gpx.timeSpan), shortFormat: true) ?? "N/A"
+                    let waypointCount = "\(gpx.wptPoints)"
+                    let gpxRow = recentlyVisibleSection.createNewRow()
+                    gpxRow.cellType = OASimpleTableViewCell.getIdentifier()
+                    gpxRow.title = title
+                    gpxRow.setObj(distance, forKey: "distance")
+                    gpxRow.setObj(time, forKey: "time")
+                    gpxRow.setObj(waypointCount, forKey: "waypointCount")
+                    gpxRow.iconName = "ic_custom_trip"
+                    gpxRow.iconTintColor = UIColor.iconColorDisabled
+                }
+            }
+        } else {
+            let noTracksSection = tableData.createNewSection()
+            let noTracksRow = noTracksSection.createNewRow()
+            noTracksRow.cellType = OALargeImageTitleDescrTableViewCell.getIdentifier()
+            noTracksRow.key = "noTracks"
+            noTracksRow.title = localizedString("no_track_files")
+            noTracksRow.descr = localizedString("import_create_track_files")
+            noTracksRow.iconName = "ic_custom_folder_open"
+            noTracksRow.iconTintColor = UIColor.iconColorDefault
+            noTracksRow.setObj(localizedString("shared_string_import"), forKey: "buttonTitle")
+        }
+    }
+    
+    override func getRow(_ indexPath: IndexPath?) -> UITableViewCell? {
+        guard let indexPath else { return nil }
+        let item = tableData.item(for: indexPath)
+        var outCell: UITableViewCell?
+        if item.cellType == OASimpleTableViewCell.getIdentifier() {
+            var cell = tableView.dequeueReusableCell(withIdentifier: OASimpleTableViewCell.getIdentifier()) as? OASimpleTableViewCell
+            if cell == nil {
+                let nib = Bundle.main.loadNibNamed(OASimpleTableViewCell.getIdentifier(), owner: self, options: nil)
```

Comment:
bad practice.
 Add in viewDidLoad
   tableView.register(UINib(nibName: OASimpleTableViewCell.reuseIdentifier, bundle: nil), forCellReuseIdentifier: OASimpleTableViewCell.reuseIdentifier). and remove  let nib = Bundle.main.loadNibNamed(